components/AnalysisForm/index.js

```diff
@@ -247,8 +237,7 @@ export default function AnalysisForm({
               <KieslerKreis
                 editMode="true"
                 analysisKey={analysisKey}
-                kieslerkreisData={kieslerkreisData}
-                setKieslerkreisData={setKieslerkreisData}
+                kieslerkreisDataset="[null,null,null,null,null,null,null,null]"
```

Comment:
Why do you need an array with so many NULLs? Is there a simpler/nicer way to handle null on the KieslerKreis component when kieslerkreisDataset is falsy? Maybe a default value for the prop with an (empty?) array?

Reply:
i just noticed i already implemented a workaround at the useState function in the kieslerkreis-component. so i can just delete this line 🙏🏻